doc_source/autoscaling.md

```diff
@@ -169,7 +169,7 @@ Complete the following steps to deploy the Cluster Autoscaler\. We recommend tha
      -p '{"spec":{"template":{"metadata":{"annotations":{"cluster-autoscaler.kubernetes.io/safe-to-evict": "false"}}}}}'
    ```
 
-1. Edit the Cluster Autoscaler deployment with the following command\.
+1. Edit the Cluster Autoscaler deployment with the following command and replace *`<YOUR CLUSTER NAME>`* with your cluster name\.
```

Comment:
Although it had mentioned in Step 2, but it would be nice to have it mentioned again here.